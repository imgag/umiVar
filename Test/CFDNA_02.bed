chr7	55259314	55259715
chr7	124392311	124392712
chr7	137029637	137030038
chr7	140452935	140453336
