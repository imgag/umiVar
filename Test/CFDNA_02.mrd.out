#MRD_log10	MRD_pval	SUM_DP	SUM_ALT	Mean_AF	Median_AF
20.0	1.00e-20	16708	3703	0.22163	0.22163
